packages/server-wallet/src/models/objective.ts

```diff
@@ -266,25 +270,27 @@ export class ObjectiveModel extends Model {
     return (await ObjectiveModel.query(tx).findByIds(objectiveIds)).map(m => m.toObjective());
   }
 
-  toObjective<O extends DBObjective = DBObjective>(): O {
-    const withParticipants = {
-      objectiveId: this.objectiveId,
-      status: this.status,
+  toObjective<O extends SupportedObjective = SupportedObjective>(): WalletObjective<O> {
+    // Reconstruct the Objective, taking on trust that we inserted it properly
+    const o = {
       type: this.type,
       data: this.data,
+      participants: [] as SupportedObjective['participants'], // reinstate an empty participants array
```

Comment:
I always found having the participants set to `[]` sometimes is kinda confusing. Could we just get rid of this? Or store it properly?

Reply:
I agree we should do one or the other. Let's make a new issue for it.

Reply:
https://github.com/statechannels/statechannels/issues/3325